x-pack/platform/plugins/shared/fleet/public/applications/fleet/sections/settings/components/edit_output_flyout/use_output_form.tsx

```diff
@@ -362,6 +363,8 @@ export function useOutputForm(onSucess: () => void, output?: Output, defaultOupu
 
   const isSSLEditable = isDisabled('ssl');
   // Logstash inputs
+  const logstashEnableSSLInput = useSwitchInput(output ? Boolean(output?.ssl) : true);
```

Comment:
The idea here is that if we don't have any pre-existing `output`, we assume its a new configuration and set the default state to true. Otherwise if we are editing an existing output, we set the enable state based on the presence of existing SSL settings. 

This is fine for editing an existing Logstash config, but if you are editing an existing output of a different type like Kafka where SSL values may not have been setup, and changing it to Logstash, it can default to false which is not ideal. We have a failing cypress test around this.

Reply:
Addressed in https://github.com/elastic/kibana/pull/216216/commits/30be1640c0749777de4f6abc6f3865529f46fdc4